src/php/PhelFacade.php

```diff
@@ -25,10 +26,14 @@ final class PhelFacade
 
     test <filename> <filename> ...
         Tests the given files. If no filenames are provided all tests in the
-        test directory are executed.
+        "tests" directory are executed.
 
     fmt <filename-or-directory> ...
         Formats the given files.
+    
+    export
```

Comment:
I'm still confused. Do we need a argument now or not?

Reply:
Nope. Why? Every time we run this command we need to remove all other generated code, the whole directory. This means if you would be able to export only one file it will be harder to keep track of what to remove and what to keep. For simplicity, I decided to don't allow parameters and let generate all exported functions at once every time. It's a simple command, where you won't need to do anything special but some previous configuration in your composer options, and that's all.

Reply:
The documentation said:
```
Or you can use the `phel export <phel-path>`
```

But you are correct we just don't need the argument. I just got confused